crates/ruff_cli/src/lib.rs

```diff
@@ -427,6 +430,11 @@ pub fn check(args: CheckCommand, log_level: LogLevel) -> Result<ExitStatus> {
             printer.write_once(&diagnostics, &mut summary_writer)?;
         }
 
+        if checked_files > 0 && diagnostics.fixed.is_empty() && diagnostics.messages.is_empty() {
+            let s = if checked_files == 1 { "" } else { "s" };
+            writeln!(summary_writer, "Found 0 errors in {checked_files} file{s}")?;
+        }
```

Comment:
Flagging the logic here on how I'm deciding whether or not to display the "success" output. Please let me know if we think the heuristic should be different!